protocol/x/clob/abci.go

```diff
@@ -180,20 +186,6 @@ func PrepareCheckState(
 		panic(err)
 	}
 
-	telemetry.ModuleSetGauge(
-		types.ModuleName,
-		float32(len(subaccountIds)),
-		metrics.Liquidations,
-		metrics.LiquidatableSubaccountIds,
-		metrics.Count,
-	)
```

Comment:
moved to `LiquidateSubaccountsAgainstOrderbook`

Reply:
You already also have this metric already on the daemon/server side right? It's kind of showing up here a 3rd time (which may be fine but may also be unnecessary)

Reply:
Currently it's on the client side (# of ids sent by daemon). there is nothing on the server side IIRC.

This effectively is the count for the server (application to be more accurate).